templates/modern-solutions/rop-demo/statement.rst

```diff
@@ -0,0 +1,3 @@
+An example of using ROP to successfully call ``checker(0x11223344)``.
```

Comment:
You accidentally the verb.

Reply:
Should I just prepend 'This task is ' to the whole sentence?